tests/transforms/test_gibbs_noise.py

```diff
@@ -25,8 +25,9 @@
 
 _, has_torch_fft = optional_import("torch.fft", name="fftshift")
 
-params = {"shape": ((128, 64), (64, 48, 80)), "input_type": TEST_NDARRAYS if has_torch_fft else [np.array]}
-TEST_CASES = list(dict_product(format="list", **params))
+shapes = ((128, 64), (64, 48, 80))
+input_types = TEST_NDARRAYS if has_torch_fft else [np.array]
+TEST_CASES = [[p_dict["shape"], p_dict["input_type"]] for p_dict in dict_product(shape=shapes, input_type=input_types)]
```

Comment:
```suggestion
TEST_CASES = list(map(list, product(shape=shapes, input_type=input_types)))
```
I think this would be equivalent if you import `itertools.product`?